adapters/airflow_adapter.py

```diff
@@ -22,8 +22,9 @@
 class AirflowAdapter(Adapter):
   """Provides a way to interact with the Airflow Database, and get data from it."""
 
-  def __init__(self, airflow_db):
+  def __init__(self, airflow_db, bucket_names):
```

Comment:
Can you use an AirflowConfig class:
```python
AirflowConfig = collection.namedtuples('AirflowConfig', ["db", "bucket_name"]) and pass this an argument.

Reply:
Since there are duplication in the bucket name and the db name, you create a method in main (when you are looking at the ENV)
```python
def parseAirflowConfig(airflow_env):
  configs = []
  for value in airflow_env.split(","):
    # do the parsing
    config.append(AirflowConfig(db, bucket))
 return configs
```